src/openzeppelin/tests/utils.cairo

```diff
@@ -1,9 +1,12 @@
-use array::ArrayTrait;
 use core::result::ResultTrait;
 use option::OptionTrait;
+use array::ArrayTrait;
```

Comment:
Small detail regarding format. I've been trying to order use clauses alphabetically, maybe there is a better rule to follow, but I think is good if we follow one (the formatted could fix this, but it doesn't atm).

Reply:
Yeah, I think that's a good approach. Will update

Reply:
FWIW i've been ordering by line length. my OCD seems to favor visual alignment over lexicographical